### PR TITLE
chore: preparing the spring

### PR DESCRIPTION
**Pull request by obdulia-losantos**
- Better handling for `VERSION` and `REVISION` files, removed `temp` folder option
- Use current `VERSION` in all modules
- Renamed `aether-common-module` to `aether-common-library`
- Renamed `aether-client` to `aether-client-library` and moved to root folder.
- Cleaned set up: setup.py files, docker-compose files, scripts...
- Better wording, the concept was `library` not `module` :nerd_face:

### CODE REVIEW

**Review by shawnsarwar — DISMISSED** [review]
Points for clarity.

**Review by shawnsarwar — DISMISSED** [review]

File: aether-couchdb-sync-module/setup.py [hunk 1]
```diff
@@ -16,14 +18,18 @@
...
+
+def read(f):
+    return open(f, 'r', encoding='utf-8').read()
+
 
 setup(
+    version=read('/var/tmp/VERSION').strip(),
     name='aether.sync',
-    description='A python module with Aether CoudhDB Sync functionality',
+    description='A python library with Aether CoudhDB Sync functionality',
```
Comment (admbtlr) [inline aether-couchdb-sync-module/setup.py]: This is actually still a module, isn't it?

File: aether-kernel/setup.py [hunk 2]
```diff
@@ -16,14 +18,18 @@
...
+
+def read(f):
+    return open(f, 'r', encoding='utf-8').read()
+
 
 setup(
+    version=read('/var/tmp/VERSION').strip(),
     name='aether.kernel',
-    description='A python module with Aether KERNEL functionality',
+    description='A python library with Aether KERNEL functionality',
```
Comment (admbtlr) [inline aether-kernel/setup.py]: Again, still a module

File: aether-odk-module/setup.py [hunk 3]
```diff
@@ -16,14 +18,18 @@
...
+
+def read(f):
+    return open(f, 'r', encoding='utf-8').read()
+
 
 setup(
+    version=read('/var/tmp/VERSION').strip(),
     name='aether.odk',
-    description='A python module with Aether ODK functionality',
+    description='A python library with Aether ODK functionality',
```
Comment (admbtlr) [inline aether-odk-module/setup.py]: Again again

File: aether-ui/setup.py [hunk 4]
```diff
@@ -16,14 +18,18 @@
...
+
+def read(f):
+    return open(f, 'r', encoding='utf-8').read()
+
 
 setup(
+    version=read('/var/tmp/VERSION').strip(),
     name='aether.ui',
-    description='A python module with Aether UI functionality',
+    description='A python library with Aether UI functionality',
```
Comment (admbtlr) [inline aether-ui/setup.py]: Again again again

**Review by admbtlr — CHANGES_REQUESTED** [review]

File: aether-couchdb-sync-module/setup.py (hunk 1 above)
Comment (obdulia-losantos) [reply aether-couchdb-sync-module/setup.py]: That's the point, from the python perspective it's a library from the aether ecosystem perspective it's a module :woman_shrugging: 

What should we do?

Comment (obdulia-losantos) [reply aether-couchdb-sync-module/setup.py]: `An aether module with CouchDB Sync functionality`?

Comment (admbtlr) [reply aether-couchdb-sync-module/setup.py]: Yes, I think that's the best option (but `Aether`, not `aether`)

File: aether-kernel/setup.py (hunk 2 above)
Comment (obdulia-losantos) [reply aether-kernel/setup.py]: A module or an app?

Comment (admbtlr) [reply aether-kernel/setup.py]: module please!

**Review by admbtlr — APPROVED** [review]
💥

**Pull request merged**